trieste/acquisition/function/function.py

```diff
@@ -232,8 +227,28 @@ def __call__(self, x: TensorType) -> TensorType:
         return (self._eta - mean) * normal.cdf(self._eta) + variance * normal.prob(self._eta)
 
 
-class log_expected_improvement(AcquisitionFunctionClass):
-    def __init__(self, model: ProbabilisticModel, eta: TensorType):
+class LogExpectedImprovement(ExpectedImprovement):
+    """
+    Builder for the log expected improvement function.
+    """
+
+    def __init__(self, search_space: Optional[SearchSpace] = None):
+        """
+        :param search_space: The global search space over which the optimisation is defined. This is
+            only used to determine explicit constraints.
+        """
+        self._search_space = search_space
+        self._acq_function_cls = log_expected_improvement
+
+    def __repr__(self) -> str:
+        """"""
+        return f"LogExpectedImprovement({self._search_space!r})"
```

Comment:
you can modify the repr to return `self.__class__.__name__` (I think its that, perhaps smth similar) in `ExpectedImprovement` and then you don't need to overwrite it this way